New UI port and 2FA (#37)
NethServer/nethsecurity-docs: remote_access.rst

Web user interface
==================

NethSecurity UI (ns-ui), the NethSecurity official web interface, is available on port ``9090`` at the following URL: **https://<server_ip>:9090**.

To ease the access, NethSecurity UI is also available on standard HTTP port ``443`` at the following URL: **https://<server_ip>** or **http://<server_fqdn>**.

Both URL are accessible from LAN and WAN by default.

Restricting access to NethSecurity UI
-------------------------------------

By default, this interface is accessible on port 9090 from both your internal network (LAN) and the wider internet (WAN).
While convenient, this can potentially introduce a security risk.

> To mitigate this risk, you can take two main steps:

Reviewer: ```suggestion
To mitigate this risk, you have two options (remove or restrict access):
```

> - remove the ``Allow-UI-from-WAN`` rule: go to the Firewall rules page,  navigate to the ``Input rules`` tab,

Reviewer: ```suggestion
- remove the ``Allow-UI-from-WAN`` rule: go to the Firewall rules page, navigate to the ``Input rules`` tab,
```

  and locate the "Allow-UI-from-WAN" rule. Click the :guilabel:`Delete` button to remove it

> - allowing access from specific IPs or networks: in the Firewall rules page, locate the "Allow-UI-from-WAN"

Reviewer: ```suggestion
- restrict access from specific IPs or networks: in the Firewall rules page, locate the "Allow-UI-from-WAN"
```

  rule and click the :guilabel:`Edit` button. In the ``Source address``  field, enter the IP addresses or network CIDRs
  from which you want to allow access to the NethSecurity UI.

>   For example, to allow access only from your home network, you could enter the IP range 192.168.1.0/24

Reviewer: ```suggestion
  For example, to allow access only from your home network, you could enter the 192.168.1.0/24 network.
```

  Only allow access from trusted IP addresses or networks. Leaving this field blank will allow anyone on the internet to access the NethSecurity UI.

Additional security measures:

- use a strong password for the admin user
- enable :ref:`two-factor authentication (2FA) <2fa-section>` for the admin user
- keep your firewall up to date with the latest security patches

Change web user interface port
------------------------------

Users have the capability to change the NethSecurity UI port.

> To change NethSecurity UI port from 9090 to 8181, execute: ::

Reviewer: ```suggestion
To change the NethSecurity UI port from 9090 to 8181, execute: ::
```

  uci set ns-ui.config.nsui_extra_port=8181
  uci commit ns-ui && ns-ui

Disable web user interface on port 443
--------------------------------------

While exposing port 443 (HTTPS) can be necessary for certain services, directly accessing the NethSecurity UI through this port

> may introduces a potential security risk. Here's how to safely maintain port 443 functionality while protecting your NethSecurity UI.

Reviewer: ```suggestion
may introduce a potential security risk. Here's how to safely maintain port 443 functionality while protecting your NethSecurity UI.
```

If you don't require accessing the NethSecurity UI through port 443, disable it to minimize attack opportunities.

> Execute the following command on your NethServer system: ::

Reviewer: ```suggestion
Execute the following commands on your NethServer system: ::
```
